commands/deploy.go

```diff
@@ -2,19 +2,91 @@ package commands
 
 import (
 	"fmt"
+	"io/ioutil"
+	"net/http"
 	"os"
+	"path/filepath"
+	"strings"
+	"time"
 )
 
-// DeployCmd handles deploying an app to Section
-type DeployCmd struct{}
+// MaxFileSize is the tarball file size allowed to be uploaded in bytes.
+const MaxFileSize = 1073741824 // 1GB
+
+// DeployCmd handles deploying an app to Section.
+type DeployCmd struct {
+	Debug     bool
+	ServerURL string `default:"https://api.section.io/code-upload/v1"`
+}
 
 // Run deploys an app to Section's edge
 func (a *DeployCmd) Run() (err error) {
```

Comment:
Just noticed that I've used `a` as the variable name for the receiver throughout all the commands. 

This should probably be named `c` (for command). 

Don't bother changing that here, and I'll submit a PR that renames the receiver everywhere.

Reply:
I will leave this for now